repair/row_level.cc

```diff
@@ -503,7 +501,9 @@ void repair_writer_impl::create_writer(lw_shared_ptr<repair_writer> w) {
     }
     replica::table& t = _db.local().find_column_family(_schema->id());
     _writer_done = mutation_writer::distribute_reader_and_consume_on_shards(_schema, std::move(_queue_reader),
-            streaming::make_streaming_consumer(sstables::repair_origin, _db, _sys_dist_ks, _view_update_generator, _estimated_partitions, _reason, is_offstrategy_supported(_reason)),
+            streaming::make_streaming_consumer(sstables::repair_origin, _db, _view_update_generator, _estimated_partitions, is_offstrategy_supported(_reason), [] (const replica::table& t) {
+                return make_ready_future<bool>(db::view::check_repair_needs_view_update_path(t));
```

Comment:
This is problematic. As I mentioned earlier, not all repair needs view update.

Reply:
Reverted in v2.